### DISCUSSION TITLE
Please add the ability to make the 'Switch Kana' key skip half-size hiragana

### DISCUSSION THREAD

**Discussion by JB0x0003** [google/mozc#771] [Ideas]
This is a feature which I personally never use and it adds a large number of button presses to my usage of this program every day. I know this is nonstandard, but it'd be incredibly useful.

**Comment by hiroyuki-komatsu** [comment]
Hi JB0x0003,

Would you tell us more information of this request?
We would like to know steps to reproduce your expected behavior with some examples?

Thank you,
